guides/common/modules/proc_keeping-track-of-your-exports.adoc

```diff
@@ -11,7 +11,7 @@ This helps you track which content was exported and to where.
 Use the `--destination-server` argument during export to indicate the target server.
 This option is available for all `content-export` operations.
 
-.Tracking destinations of Library exports
+.Tracking destinations of library exports
```

Comment:
```suggestion
.Tracking destinations of Library exports
```
We capitalize the ["Library" environment](https://github.com/theforeman/foreman-documentation/blob/master/CONTRIBUTING.md#capitalization)

Reply:
Will fix